rubrik_cdm/rubrik_cdm.py

```diff
@@ -513,6 +513,23 @@ def status(self, request_id="1", timeout=15):
 
         return api_request
 
+    def discover(self, timeout=30):
+        """Searches for nodes bootstrappable to the specified Rubrik cluster
+
+        Keyword Arguments:
+            timeout {int} -- The response timeout value, in seconds, of the API call. (default: {30})
+
+        Returns:
+            dict -- The response returned by `GET /internal/cluster/me/discover`.
+        """
+
+        self.log('status: Discovering bootstrappable nodes for the Rubrik Cluster.')
+        discover_api_endpoint = '/cluster/me/discover'
```

Comment:
We don't need to define `discover_api_endpoint` as a separate variable. It can be stored as a string directly in the `get()`